src/main/java/com/kodilla/ecommercee/mapper/GroupMapper.java

```diff
@@ -14,11 +13,9 @@
  */
 @Component
 public class GroupMapper {
-    private ProductService productService;
     private ProductMapper productMapper;
 
-    public GroupMapper(ProductService productService, ProductMapper productMapper) {
-        this.productService = productService;
+    public GroupMapper(ProductMapper productMapper) {

```

Comment:
Czemu dajesz tutaj konstruktor? Nie lepiej zrobić wstrzykiwanie @Autowired przez parametr? Wiem, że przez parametr jest niezalecane, ale zastanawia mnie, dlaczego takie rozwiązanie wybrałeś. :-)

Reply:
http://olivergierke.de/2013/11/why-field-injection-is-evil/